js/webcomponent/api-updates-subscribe/main.js

```diff
@@ -49,6 +49,15 @@ class ApiSubscribe extends HTMLElement {
     if (closeModalBtn) {
       closeModalBtn.focus()
     }
+
+    // Log to SiteImprove when opening the modal
+    if(typeof SiteImprove !== 'undefined'){
+      SiteImprove.registerEvent({
+        category: 'Flow steps',
```

Comment:
We will update the documentation in MDS so we use singular form on the predefined categories:

```suggestion
        category: 'Flow step',
```